base/test/middleware/jwt_test.mustache

```diff
@@ -0,0 +1,37 @@
+const fetch = require('node-fetch');
+const jwt   = require('jsonwebtoken');
```

Comment:
Ah, sorry my bad. I normally align things in my projects, must have done this accidentally here. Will put this back